sdk/template-LLC/Azure.Template.LLC/api/Azure.Template.LLC.netstandard2.0.cs

```diff
@@ -0,0 +1,25 @@
+namespace Azure.Template.LLC
+{
+    public partial class TemplateServiceClient
```

Comment:
Do we need this file in template?

Reply:
I think it's good to see how a fully generated solution works, but interested to hear other perspectives - what would be a reason not to include this file in the template?